plugin/clone/include/clone_hton.h

```diff
@@ -75,6 +75,10 @@ struct Locator {
       /* Should not lock plugin for auxiliary threads */
       assert(thd != nullptr);
 
+      DBUG_EXECUTE_IF(
+          "client_has_more_se",
+          if (m_hton->db_type == DB_TYPE_ROCKSDB) db_type = DB_TYPE_UNKNOWN;);
```

Comment:
m_hton is going to be nullptr here. I think the check should be `db_type == DB_TYPE_ROCKSDB`